distribution/docker/src/docker/bin/docker-entrypoint.sh

```diff
@@ -73,6 +73,10 @@ if [[ -n "$ES_LOG_STYLE" ]]; then
   esac
 fi
 
+if [[ -d /usr/local/cloudflare-zlib/lib ]]; then
+  export LD_LIBRARY_PATH=/usr/local/cloudflare-zlib/lib
```

Comment:
should we try to preserve existing values of `LD_LIBRARY_PATH`, e.g.
```suggestion
  export LD_LIBRARY_PATH=/usr/local/cloudflare-zlib/lib:$LD_LIBRARY_PATH
```

Reply:
I wondered about that, but I also struggled to think of another reason why someone else would also be setting `LD_LIBRARY_PATH` with our image. Any ideas, anyone?

Also, I now think that this isn't the right place for this change, because Cloud do something different with the Docker entrypoint. I'll move this into the `elasticsearch` script and put a guard on it with `$ES_DISTRIBUTION_TYPE`.